src/src/PluginStructs/P093_data_struct.h

```diff
@@ -170,6 +172,7 @@ struct P093_data_struct : public PluginTaskData_base {
       vane(0),
       wideVane(0),
       roomTemperature(0),
+      remoteTemperature(0),
```

Comment:
This could be initialized with a float zero value: `0.0f` (can also be applied to the other 2 float values `temperature` and `roomTemperature`)